datafusion/core/tests/expr_api/mod.rs

```diff
@@ -167,21 +167,6 @@ fn test_list_range() {
     );
 }
 
-#[tokio::test]
-async fn test_aggregate_error() {
-    let err = first_value_udaf()
-        .call(vec![col("props")])
-        // not a sort column
-        .order_by(vec![col("id").sort(true, false)])
-        .build()
-        .unwrap_err()
-        .to_string();
-    assert_contains!(
-        err,
-        "Error during planning: ORDER BY expressions must be Expr::Sort"
-    );
-}
```

Comment:
test case removed because the API now makes it impossible to provide wrong expression type